Sources/AppStoreConnectCLI/Commands/TestFlight/TestFlightCommand.swift

```diff
@@ -9,6 +9,7 @@ public struct TestFlightCommand: ParsableCommand {
         abstract: "Manage your beta testing program, including beta testers and groups, apps, and builds.",
         subcommands: [
             TestFlightAppsCommand.self,
+            TestFlightBetaGroupCommand.self,
```

Comment:
```suggestion
            TestFlightBetaGroupsCommand.self,
```

Reply:
I am confusing myself a little with pluralisation. We call the command as `asc testflight betagroup` so I think no pluralisation here is fine.
Maybe I should remove it from everything. e.g. `BetaGroupListCommand` starts to sound more like a noun. It's the: beta group list